rpm_spec/manageiq.spec.in

```diff
@@ -25,25 +29,27 @@ Summary:  %{product_summary}
 License:  Apache-2.0
 URL:      %{product_url}
 
-Source0:  %{name}-appliance-%{version}.tar.gz
-Source1:  %{name}-core-%{version}.tar.gz
-Source2:  %{name}-gemset-%{version}.tar.gz
-Source3:  %{name}-manifest-%{version}.tar.gz
+Source0:  %{name}-ansible-venv-%{version}.tar.gz
+Source1:  %{name}-appliance-%{version}.tar.gz
+Source2:  %{name}-core-%{version}.tar.gz
+Source3:  %{name}-gemset-%{version}.tar.gz
+Source4:  %{name}-manifest-%{version}.tar.gz
 
 %description
 %{product_summary}
 
 %prep
 umask 0002
-%setup -q -n %{appliance_builddir}
-%setup -q -T -D -b 1 -n %{core_builddir}
-%setup -q -T -D -b 2 -n %{gemset_builddir}
-%setup -q -T -D -b 3 -n %{manifest_builddir}
+%setup -q -T -b 0 -n %{ansible_venv_builddir}
+%setup -q -T -b 1 -n %{appliance_builddir}
+%setup -q -T -b 2 -n %{core_builddir}
+%setup -q -T -b 3 -n %{gemset_builddir}
+%setup -q -T -b 4 -n %{manifest_builddir}
 
-# buildsubdir is set to the last extracted archive (gemset), reset
+# buildsubdir is set to the last extracted archive, reset
 cd %{_builddir}
 
-pathfix.py -pni "%{__python3} %{py3_shbang_opts}" %{gemset_builddir}
+%py3_shebang_fix %{gemset_builddir}
```

Comment:
I _think_ we can remove this line entirely, but I'd rather do that in a follow up.  Even so, I switched to the dedicated macro for this task.